File: backend/scripts/hello.py
#!/usr/bin/env python3
"""
Hello World Script

A simple hello world script for the NanoBanana project.
"""

import sys


def greet(name: str = "World") -> str:
    """Return a greeting message."""
    return f"Hello, {name}!"


def main() -> None:
    """Main entry point for the script."""
    name = sys.argv[1] if len(sys.argv) > 1 else "World"

    print(greet(name))
    print("\nWelcome to NanoBanana - Simple API for AI image generation!")


if __name__ == "__main__":
    main()

File: hello.ts
/**
 * Hello World Script
 *
 * A simple hello world script for the NanoBanana project.
 */

function greet(name: string = "World"): string {
  return `Hello, ${name}!`;
}

function main(): void {
  const args = process.argv.slice(2);
  const name = args[0];

  console.log(greet(name));
  console.log("\nWelcome to NanoBanana - Simple API for AI image generation!");
}

// Run if called directly
if (require.main === module) {
  main();
}

export { greet };
